.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
We don't need any of the files from openstreetmap-carto for building the containers. This avoids that those files are included in the build context. See https://docs.docker.com/engine/reference/builder/#dockerignore-file.